dbt_project.yml

```diff
@@ -326,31 +286,11 @@ models:
       polygon:
         +schema: tigris_polygon
 
-    x2y2:
-      +schema: x2y2
-      ethereum:
-        +schema: x2y2_ethereum
-    zora:
-      +schema: zora
-      ethereum:
-        +schema: zora_ethereum
-    magiceden:
-      +schema: magiceden
-      polygon:
-        +schema: magiceden_polygon
-      solana:
-        +schema: magiceden_solana
-
     nomad:
       +schema: nomad
       ethereum:
         +schema: nomad_ethereum
 
-    ellipsis_finance:
```

Comment:
this one a mistake?

Reply:
no, it was defined double in the `dbt_project` file.